auction-house/program/src/lib.rs

```diff
@@ -1388,6 +1498,29 @@ pub struct AuctionHouse {
 
 pub const TRADE_STATE_SIZE: usize = 1;
 
+pub const RECEIPT_SIZE: usize = 8 + //key
+32 + // trade_state
+32 + // bookkeeper
+32 + // auction_house
+32 + // wallet
+32 + // token_account
+8 + // price
+8 + // token_size
+1; // bump 
+
+
+#[account]
+pub struct Receipt {
+    pub trade_state: Pubkey,
+    pub bookkeeper: Pubkey,
+    pub auction_house: Pubkey,
+    pub wallet: Pubkey,
+    pub token_account: Pubkey,
```

Comment:
Not tracking the metdata address for nft associated to the receipt because it can be derived from the mint of the token_account.

Reply:
https://github.com/metaplex-foundation/metaplex-program-library/blob/master/token-metadata/program/src/pda.rs#L29